zipline/finance/blotter.py

```diff
@@ -179,64 +185,62 @@ def process_benchmark(self, benchmark_event):
         return
         yield
 
-    def process_trade(self, trade_event):
+    def process_open_orders(self):
```

Comment:
I guess we could make everything in blotter that needs to know the current simulation time takes that as a parameter.  I'll look into that.